quick-start.rst

```diff
@@ -249,12 +249,12 @@ You can then use a text editor like vim or nano to go in there and fill out the
 Extra Steps for EE
 -----------------------------------------------------------------------------------------------------
 
-1. Log into https://dashboard.license.netmaker.io"
-2. Copy License Key Value: https://dashboard.license.netmaker.io/license-keys"
-3. Retrieve Account ID: https://dashboard.license.netmaker.io/user"
-4. Place the licence key and account ID in the netmaker.env file.
+1. Log into our SAAS platform at https://app.netmaker.io" and create an account.
+2. make a self-hosted tenant. details in our :doc:`Netmaker Enterprise setup <./ee/ee-setup>`
```

Comment:
1. Visit `https://app.netmaker.io` to create your account on the Netmaker SaaS platform.
2. Create a tenant of type `self-hosted` to obtain a license key